util/util_test.go

```diff
@@ -18,27 +18,27 @@ func TestIsIP(t *testing.T) {
 	assert.False(t, IsIP(notIP))
 }
 
-func TestFileExists(t *testing.T) {
-	filePath := "./.jeinwei8380243unt4u"
-	os.Remove(filePath)
-	file, err := os.OpenFile(filePath, os.O_RDONLY|os.O_CREATE, 0666)
-	assert.Nil(t, err)
-	file.Close()
-	exists, err := Exists(filePath)
-	assert.Nil(t, err)
-	assert.True(t, exists)
-	os.Remove(filePath)
-	exists, err = Exists(filePath)
-	assert.Nil(t, err)
-	assert.False(t, exists)
+// func TestFileExists(t *testing.T) {
```

Comment:
Commented this test out because it relies on the filesystem and doesn't make a great unit test like the rest of the tests in this file.